build.gradle

```diff
@@ -46,12 +52,12 @@ compileTestKotlin {
 }
 
 dependencies {
-    api platform('com.exactpro.th2:bom:4.0.2')
+    api platform('com.exactpro.th2:bom:4.1.0')
 
     api 'com.exactpro.th2:grpc-common:3.11.1'
 
     implementation 'com.google.guava:guava'
-    implementation 'io.github.microutils:kotlin-logging:2.1.23'
+    implementation 'io.github.microutils:kotlin-logging:3.0.2'
```

Comment:
This version depends on kotlin 1.7. But we use kotlin 1.6. Do we have to update this version? If we do please make sure that in the result library we depend on the kotlin 1.6